### PR TITLE
Update cluster-administration flow-control.md and system-metrics.md

### PR DESCRIPTION
**Pull request by ydFu**
* sync with english version in cluster-administration flow-control.md and system-metrics.md

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -938,8 +938,10 @@ You can fetch like this:
...
 You can make suggestions and feature requests via
-[SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+[SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery)
+or the feature's [slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
 -->
 有关API优先级和公平性的设计细节的背景信息，
 请参阅[增强建议](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md)。
-你可以通过 [SIG APIMachinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) 提出建议和特性请求。
+你可以通过 [SIG APIMachinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) 提出建议和特性请求，
+或特性的[slack 频道](http://kubernetes.slack.com/messages/api-priority-and-fairness)。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/flow-control.md:947]:
```suggestion
你可以通过 [SIG APIMachinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery)
或特性的 [Slack 频道](http://kubernetes.slack.com/messages/api-priority-and-fairness)
提出建议和特性请求。
```

Comment (ydFu) [reply content/zh/docs/concepts/cluster-administration/flow-control.md:947]: thank you for suggesting me.

**Pull request merged**